.github/workflows/cd.yml

```diff
@@ -145,19 +172,22 @@ jobs:
     needs:
       - setup
     with:
-      branch: ${{ github.event.inputs.branch }}
-      run-playwright: ${{ github.event.inputs.run-playwright == true }}
+      branch: ${{ inputs.branch }}
+      go-version: ${{ inputs.go-version }}
+      node-version: ${{ inputs.node-version }}
+      golangci-lint-version: ${{ inputs.golangci-lint-version }}
+      run-playwright: ${{ inputs.run-playwright == true }}
```

Comment:
is this 

`${{ inputs.run-playwright == true }}`

somehow different from just

`${{ inputs.run-playwright }}` ?

Reply:
I *think* I had to switch it to `${{ inputs.run-playwright == true }}` otherwise it would not be considered a boolean and it caused weirdness somewhere else down the line.

It was already like this before the PR.

Reply:
Changed after Sam's comment, I think we can safely get rid of those if we use `inputs` instead of `github.event.inputs` :)